controller/search.js

```diff
@@ -17,6 +17,13 @@ function setup( config, backend, query ){
       return next();
     }
 
+    // do not run controller if there are already results
+    // this was added during libpostal integration.  if the libpostal parse/query
+    // doesn't return anything then fallback to old search-engine-y behavior
+    if (res && res.hasOwnProperty('data') && res.data.length > 0) {
+      return next();
+    }
+
```

Comment:
would be great to add some logging here to let us know that we used the original after all, so we can do some analysis on how often this happens and for which queries. that way we can later determine if turning it off is the better way to go. also would be good to set a flag of some sort in the request or result objects so that we can use it in confidence score computation. generally if we used this `original` behavior it's safe to assume that confidence in the results will be lower than usual.